### ISSUE TITLE
Replace property literals with `Properties` constants

### ISSUE THREAD

**Issue by sbespalov** [strongbox/strongbox#1842]
# Task Description
All the vertices and edges property literals should be replaced with corresponding constant from `org.carlspring.strongbox.db.schema.Properties`. This should be fixed mostly in `org.carlspring.strongbox.gremlin.adapters` package.

# Task Relationships

This task:
* Is a sub-task of: #1649 

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @sbespalov
  * @steve-todorov

**Comment by sreejadas** [comment]
Hi, I am one of the mentors for Grace Hopper OSD. Can I work on this?

**Comment by carlspring** [comment]
@sreejadas ,

Sorry, I've only just seen your message! Yes, please feel free to look into this. Please, note that the work for this should be done against the `issues/1649` branch.

**Comment by liooh** [comment]
Hi, I've interest in work in this task, however. @sreejadas has some update?

**Comment by carlspring** [comment]
Hi @liooh ,

I don't think she has done any work on this! Feel free to pick it up! :)

**Comment by liooh** [comment]
Ok, thanks!

**Comment by liooh** [comment]
Ok, I will start the development

**Comment by carlspring** [comment]
Okay, @liooh ! Your point of contact for this will be @sbespalov .

**Comment by liooh** [comment]
Should Edges.java and Vertices.java files be removed?

**Comment by sbespalov** [comment]
no, you should use constants defined there instead of hardcoded literals within all the code.

**Comment by liooh** [comment]
Ok
